tee-worker/app-libs/stf/src/trusted_call.rs

```diff
@@ -787,9 +804,7 @@ where
 				IMTCall::set_identity_networks { who, identity, web3networks }
 					.dispatch_bypass_filter(ita_sgx_runtime::RuntimeOrigin::root())
 					.map_err(|e| Self::Error::Dispatch(format!(" error: {:?}", e.error)))?;
-				let res = SetIdentityNetworksResponse { req_ext_hash: hash };
```

Comment:
I think we need to keep this here for compatibility until clients (i.e. IDHub) are no longer relying on it to detect the response, no?

Reply:
Well if the tests are passing that seems to indicate that the Old Way (ie listening for the response on the parachain events) still works, no? :3

Reply:
In terms of response channel - yes it works as before. 
Currently client can:

- in case of II listen to parachain events
- in case of DI wait for response on WS or listen to parachain events